satzung.rst

```diff
@@ -97,6 +97,9 @@ Der StAPF ist an die Weisungen des Plenums gebunden, kann jedoch
 eigenverantwortlich handeln und muss seine Beschlüsse dem ZaPF-Plenum gegenüber
 vertreten.
 Die Entscheidungen innerhalb des StAPF müssen in diesen Fällen einstimmig fallen.
+Der StAPF ist beschlussfähig falls mindestens drei seiner Mitglieder auf einer
+Sitzung anwesend sind und der Beschluss in der Sitzungseinladung angekündigt
+wurde.
```

Comment:
Wenn ich gerade nichts grob übersehe war der letzte echte Beschluss des StAPF ein Notvorschlag für die Entsendung in den SysAkkPool auf dem PVT in Dresden. Und da war es auch absehbar, dass wir die Entsendung vornehmen müssen, konnten also damit einladen. Ich denke daher dass das kein Problem für die Zukunft sein sollte.